Makefile

```diff
@@ -46,7 +46,7 @@ install: publish-local ## Install the plugin binaries to the local machine
 
 .PHONY: build-local
 build-local: prepare test ## Build the plugin binaries locally for the host architecture
-	@echo BUILD_VERSION: $(BUILD_VERSION)
+    echo BUILD_VERSION: $(BUILD_VERSION);
```

Comment:
does the `echo` works in linux/windows/mac without the `@`?

Reply:
Below screens from linux and windows runner as well as my mac machine where I ran make commands to see if it picks up the right version and it seems to me working.

linux 
![image](https://user-images.githubusercontent.com/108013945/196515568-a0f39458-a0bb-469b-a246-566ad944ec9b.png)

Wndows
![image](https://user-images.githubusercontent.com/108013945/196515654-df29bdbc-1bc0-48a3-b4f3-1d95c98eeee8.png)

mac os 
![image](https://user-images.githubusercontent.com/108013945/196515688-9006a35a-3467-4661-b698-be182ae83660.png)

Reply:
As the `@` is intended to not _echo_ the output of the commend, i see value removing the `@`